CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+**Contributions are welcome.**
```

Comment:
What sorts?

Reply:
I'd rather not limit peoples' imagination. People will have an idea what they want to do by the time they get to this document, and if someone wants to contribute an introduction poem, I'm happy to review metre.